### ISSUE TITLE
Flake8 Crashed!

### ISSUE THREAD

**Issue by MindlessHand** [AtomLinter/linter-flake8#583]
Flake8 Crashed!
linter-flake8:: Flake8 threw an error related to:
AttributeError: 'FlakesChecker' object has no attribute 'JOINEDSTR'
Please check Atom's Console for more details

The code snippet what caused this crash:
x = "end"
print(f"This will be the {x}")
I think the problem is with the support of the new fstring literals.
If i remove this part of the code then everything works fine.

Debug information:
Atom version: 1.25.0
Flake8 version: `2.5.5 (pep8: 1.5.7, mccabe: 0.2.1, ProxyChecker: 0.0.1, hacking.core: 0.0.1, pyflakes: 0.8.1, flake8-docstrings: 1.3.0, pydocstyle: 2.1.1) CPython 3.6.5 on Windows`

**Comment by Arcanemagus** [comment]
Are you able to run `flake8` manually on that file? The error is coming from `flake8`'s `stderr` stream.

**Comment by MindlessHand** [comment]
Unfortunatelly i'm not able to do it.
Got the following error msg:
AttributeError: 'FlakesChecker' object has no attribute 'JOINEDSTR'

**Comment by MindlessHand** [comment]
Yeah...you were right. The outdated hacking.core caused my problem.
Thank you for your help.
Have a nice day.
